src/tests/test_connection.py

```diff
@@ -1,15 +1,37 @@
 import sys
 from unittest.mock import Mock
 
+import pytest
 from sqlalchemy.engine import Engine
 
 from sql.connection import Connection
 
 
-def test_password_isnt_displayed(monkeypatch):
+@pytest.fixture
+def cleanup():
+    yield
+    Connection.connections = {}
+
+
+@pytest.fixture
+def mock_postgres(monkeypatch, cleanup):
     monkeypatch.setitem(sys.modules, "psycopg2", Mock())
     monkeypatch.setattr(Engine, "connect", Mock())
 
+
+def test_password_isnt_displayed(mock_postgres):
     Connection.from_connect_str("postgresql://user:topsecret@somedomain.com/db")
 
     assert "topsecret" not in Connection.connection_list()
+
+
+def test_connection_name(mock_postgres):
```

Comment:
Not testing list and set alias?

Reply:
I'm testing it, see the `test_close_connection_with_alias` test